README.md

```diff
@@ -29,8 +29,9 @@ const App = () => {
           <ArcherElement
             id="root"
             relations={[{
-              from: { anchor: 'bottom' },
-              to: { anchor: 'top', id: 'element2' }
+              id: 'element2',
```

Comment:
I'd rather have `targetId`. It's obvious but still makes it more readable at first glance.

Reply:
I like it better as well :)